Cannot find module '../util'

Just installed as a local dependency on a brand new project with nothing else in it and I get an error when trying to run artillery dino

> artillery dino

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '../util'

I'm running node 12.18.2, npm 6.14.5 on macOS Catalina 10.15.7. No other packages are installed on this project and I followed the documentation here: https://artillery.io/docs/guides/getting-started/installing-artillery.html

--- a/package.json
+++ b/package.json
@@ -8,7 +8,11 @@
   },
   "files": [
     "/bin",
-    "/lib"
+    "/lib",
+    "/core",
+    "console-reporter.js",
+    "util.js",
+    ".artilleryrc"
   ],
   "oclif": {
     "commands": "./lib/cmds",

